book/src/SUMMARY.md

```diff
@@ -54,13 +54,13 @@
   * [Merge Migration](./merge-migration.md)
   * [Late Block Re-orgs](./late-block-re-orgs.md)
   * [Blobs](./advanced-blobs.md)
-* [Built-In Documentation](./help_general.md)
+* [Command Line Reference (CLI)](./help_general.md)
```

Comment:
Btw the heading for the doc is `Lighthouse General Commands`, do we want to make it match the link too?

Reply:
good idea!

I changed it, but it doesn't match exactly because I thought it would be nice to keep "Lighthouse" in the page title

Reply:
a6540f9d67332174e0df4e69e93c7dc331635a9d